### DISCUSSION TITLE
How to register CoinStore on TestNet/Mainnet

### DISCUSSION THREAD

**Discussion by 0xblockbard** [aptos-labs/aptos-developer-discussions#449] [Questions] [answered]
Hello, I faced this error "Account hasn't registered `CoinStore` for `CoinType`" when trying to send AptosCoin to my module deployed on testnet. 

For unit testing we are able to do this "coin::register<AptosCoin>(signer)", is there a way to register CoinStores on testnet/mainnet as well?

**Comment by kshitijc1506** [comment, marked as answer]
Hey you can call into [`aptos_account::deposit_coins`](https://github.com/aptos-labs/aptos-core/blob/51f896c2c71f98c0a5e3e8203ea7b8eefe5477ca/aptos-move/framework/aptos-framework/sources/aptos_account.move#L116) to register without the signer.

**Reply by 0xblockbard** [reply]
Hey thanks for the prompt reply! Is there a way to do it through the block explorer instead of through a smart contract call?

**Reply by kshitijc1506** [reply]
Yes, you can visit 0x1 on the explorer to access the framework modules.

**Reply by 0xblockbard** [reply]
Thanks! However I don't seem to be able to transfer to a module to register its CoinStore. 
I've republished the module after editing the code instead to fix the CoinStore registering issue so it works now!
